### PR TITLE
Using ES6 and modules for awesome books

### PR DESCRIPTION
**Pull request by FrankMugagga**
Created a new directory
Used npm init -y command to create package.json file.
Re-created awesome books with the  index.js, index.css file index.html file. 
Created a directory called modules and put there the modules.

Used ES6 syntax to
Divide code into modules.
Created a separate file for each module in the modules directory.
Imported modules into the index.js file. 
Refactored the functions to arrow functions.
Display the current date below the navigation bar.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line import/extensions
+import Book from './modules/book.js';
+// eslint-disable-next-line import/extensions
+import BookManager from './modules/bookManager.js';
+// eslint-disable-next-line import/extensions
+import displayBooks from './modules/displayBooks.js';
+// eslint-disable-next-line import/extensions
```
Comment (Ghiftee) [inline index.js]: - [ ] Let's try to no disable linters as they help us write cleaner code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <!--
+    <form id="bookForm">
+      <label for="title">Title:</label>
+      <input type="text" id="title" name="title" required>
+      <label for="author">Author:</label>
+      <input type="text" id="author" name="author" required>
+      <button type="submit">Add Book</button>
+    </form>
+
+    <ul id="booksList"></ul>-->
```
Comment (Ghiftee) [inline index.html]: - [ ] Kindly remove commented code as this helps your codebase to be cleaner

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @FrankMugagga  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ No linter errors
✔️ Code is divided into modules

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @FrankMugagga 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**